backend/Dockerfile

```diff
@@ -90,4 +90,5 @@ RUN sed -E "s#/(blob|tree)/master/#/\1/${COMMIT_SHA}/#g" -i /config/sample_confi
 EXPOSE 8888
 
 # Start the apiserver
+ENV MAX_NUM_WORKFLOWS 500
```

Comment:
discussion:

1. shall we provide a larger default?
e.g. we verified 7000+ workflows can still function properly in a cluster

2. maybe `MAX_NUM_FINISHED_WORKFLOWS`? to make it clear it won't affect running workflows